### PR TITLE
Docusaurus

### PR DESCRIPTION
**Pull request by jmcook1186**
### Types of changes

- [x] Enhancement (project structure, spelling, grammar, formatting)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### A description of the changes proposed in the Pull Request

Migrates all documentation to docusaurus project in `website` folder. 
View docs website by navigating to `docs` and running `npm run start`

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,4 @@ dist
 yarn-error.log
 build
 coverage
+excalidraw
```
Comment (narekhovhannisyan) [inline .gitignore:12]: newline

File: package.json [hunk 2]
```diff
@@ -51,4 +51,4 @@
   },
   "stability": "stable",
   "types": "src/index.d.ts"
-}
+}
```
Comment (narekhovhannisyan) [inline package.json:54]: newline

File: website/src/pages/index.module.css [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  .heroBanner {
+    padding: 2rem;
+  }
+}
+
+.buttons {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
```
Comment (narekhovhannisyan) [inline website/src/pages/index.module.css:23]: newline

**Comment by narekhovhannisyan** [comment]
@jmcook1186 @jawache why are not keeping documentation in the separate repository?

**Comment by jmcook1186** [comment]
We could break docs out into a separate repo, however it means we will have to monitor two independent issue/pr boards and remember to raise separate PRs across two repos for docs changes associated with changes to source code.
Neither of these points are blockers - we should discuss - but I personally favour keeping the monorepo structure for the alpha release.

**Comment by jmcook1186** [comment]
closing in favour of #239

**Pull request closed without merging**